src/features/student/views/StudentToolsTracesView/components/StudentToolsTracesViewTabs/StudentToolsTracesViewTabs.test.ts

```diff
@@ -0,0 +1,93 @@
+import type { TracesSummaryDTO } from '@/api/avenir-esr'
+import StudentToolsTracesViewTabs from '@/features/student/views/StudentToolsTracesView/components/StudentToolsTracesViewTabs/StudentToolsTracesViewTabs.vue'
+import { QueryClient, VueQueryPlugin } from '@tanstack/vue-query'
+import { flushPromises, mount, RouterLinkStub, type VueWrapper } from '@vue/test-utils'
+import { BddTest } from 'tests/utils'
+import { afterEach, beforeEach, expect, vi } from 'vitest'
+
+const tracesSummaryRef = ref<TracesSummaryDTO | undefined>({ associated: 12, unassociated: 5, totalWarnings: 2, totalCriticals: 1 })
+vi.mock('@/features/student/queries', () => ({
+  useTracesSummaryQuery: () => ({ data: tracesSummaryRef }),
+}))
+
+BddTest().given('StudentToolsTracesViewTabs', () => {
```

Comment:
```suggestion
BddTest().given('a StudentToolsTracesViewTabs', () => {
```
On dirait que la règle lint s'est perdue ici aussi mais ça devait pas commencer par des majuscules, je sais pas pourquoi